packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -38,19 +38,18 @@
     overflow-y: auto;
     padding-bottom: 60px;
     padding-top: var(--terra-consumer-nav-padding-top, 25px);
+    position: relative;
```

Comment:
interesting, you just needed to add relative positioning? Does this fix it in IE too?

Reply:
Yes, win10 Edge and IE
![image](https://user-images.githubusercontent.com/17807360/30224194-cc414e62-949b-11e7-80f9-7e292ad930ce.png)

Reply:
yes, win10 Edge and IE. I think height: 100% which is normal position but when position is set to relative, nav behaves w.r.t to normal position.